libraries/Zigbee/examples/Zigbee_Color_Dimmable_Light/Zigbee_Color_Dimmable_Light.ino

```diff
@@ -98,6 +102,8 @@ void loop() {
         Zigbee.factoryReset();
       }
     }
+    // Increase blightness by 50 every time the button is pressed
+    zbColorLight.setLightLevel(zbColorLight.getLightLevel() + 50);
```

Comment:
Could it overflow?

Reply:
the setLightLevel parameter is uint8_t so max is 255. When it overflows, it's starting from 0 + the overflowed value.
Its like that on purpose to have it simple :)